app/src/main/java/com/p2p/presentation/truco/actions/TrucoActionsBottomSheetFragment.kt

```diff
@@ -105,18 +148,36 @@ class TrucoActionsBottomSheetFragment : BaseBottomSheetDialogFragment<ViewTrucoA
 
     private fun onCollapsed() {
         binding.openButtonIcon.animateRotation(0f)
-        changeVisibleButtonsEnable(false)
+        disableEnvidoAndTrucoButtons()
     }
 
     private fun onExpanded() {
         binding.openButtonIcon.animateRotation(180f)
-        changeVisibleButtonsEnable(true)
+        enableEnvidoAndTrucoButtons()
```

Comment:
Y esto sería simplemente

```suggestion
        changeVisibleButtonsEnable(!envidoDisabledForHand, !trucoDisabledForHand)
```

Reply:
no, cómo, la idea de poder deshabilitarlos para toda la mano es que justamente el onExpanded y onColapsed no me cambie esos valores, si no yo deshabilito el envido porque lo cantaron y en el onExpanded me lo va a poner en true. Para eso están el `envidoDisabledForHand` y el `trucoDisabledForHand`

Reply:
No entiendo, porque justamente es lo que se hace acá lo que vos decís